tests/test_wrappers.py

```diff
@@ -0,0 +1,251 @@
+    ref = dataframe_na.copy()
+    ref[variables_to_impute] = ref[variables_to_impute].fillna(-999)
+
+    dataframe_na_transformed = transformer.fit_transform(dataframe_na)
+
+    # init params
+    assert isinstance(transformer.transformer, SimpleImputer)
+    assert transformer.variables == variables_to_impute
+    # fit params
+    assert transformer.input_shape_ == (8, 6)
+    # transformed output
+    assert all(dataframe_na_transformed[na_variables_left_after_imputation].isna().sum() != 0)
+    assert all(dataframe_na_transformed[variables_to_impute].isna().sum() == 0)
+    pd.testing.assert_frame_equal(ref, dataframe_na_transformed)
+
+
+def test_sklearn_imputer_object_with_constant(dataframe_na):
+    variables_to_impute = ['Name', 'City']
+    na_variables_left_after_imputation = [col
+                                          for col
+                                          in dataframe_na.loc[:, dataframe_na.isna().any()].columns
+                                          if col not in variables_to_impute]
+
+    transformer = SklearnTransformerWrapper(transformer=SimpleImputer(fill_value='missing', strategy='constant'),
+                                            variables=variables_to_impute)
+
+    # transformed dataframe
+    ref = dataframe_na.copy()
+    ref[variables_to_impute] = ref[variables_to_impute].fillna('missing')
+
+    dataframe_na_transformed = transformer.fit_transform(dataframe_na)
+
+    # init params
+    assert isinstance(transformer.transformer, SimpleImputer)
+    assert transformer.variables == variables_to_impute
+    # fit params
+    assert transformer.input_shape_ == (8, 6)
+    # transformed output
+    assert all(dataframe_na_transformed[na_variables_left_after_imputation].isna().sum() != 0)
+    assert all(dataframe_na_transformed[variables_to_impute].isna().sum() == 0)
+    pd.testing.assert_frame_equal(ref, dataframe_na_transformed)
+
+
+def test_sklearn_imputer_allfeatures_with_constant(dataframe_na):
+    transformer = SklearnTransformerWrapper(transformer=SimpleImputer(fill_value='missing', strategy='constant'))
+
+    # transformed dataframe
+    ref = dataframe_na.copy()
+    ref = ref.fillna('missing')
+
+    dataframe_na_transformed = transformer.fit_transform(dataframe_na)
+
+    # init params
+    assert isinstance(transformer.transformer, SimpleImputer)
+    # fit params
+    assert transformer.input_shape_ == (8, 6)
+    # transformed output
+    assert all(dataframe_na_transformed.isna().sum() == 0)
+    pd.testing.assert_frame_equal(ref, dataframe_na_transformed)
+
+
+def test_sklearn_standardscaler_numeric(dataframe_vartypes):
+    variables_to_scaling = ['Age', 'Marks']
+    transformer = SklearnTransformerWrapper(transformer=StandardScaler(), variables=variables_to_scaling)
+
+    ref = dataframe_vartypes.copy()
+    ref[variables_to_scaling] = (ref[variables_to_scaling] - ref[variables_to_scaling].mean()) / ref[
+        variables_to_scaling].std(ddof=0)
```

Comment:
what is ddof=0?

Reply:
ddof is degrees of freedom, but now I even don't know why I've put this parameter, especially that this is default for `std` :) Will remove this in next commit to make this not confusing

Reply:
I've checked this once again - I was wrong, `std` method in pandas dataframe has default for ddof parameter equal 1, in contrary to `StandardScaler` transformer which docs state that they are using `numpy.std(x, ddof=0)`. I'll stay with ddof=0 as it is testing sklearn transformer...